cooling/scripts/fig_heating.py

```diff
@@ -25,17 +25,21 @@ def compute_energy(ensemble, B_z, omega, theta, kx, ky, kz):
             ensemble.ensemble_properties['charge'])
     return energy
 
+# TODO: Generate a zero temperature initial state using the mode_analysis code.
+my_ensemble.v[:, 0] = -mode_analysis.wrot * my_ensemble.x[:, 1]
+my_ensemble.v[:, 1] =  mode_analysis.wrot * my_ensemble.x[:, 0]
+my_ensemble.v[:, 2] *= 0
```

Comment:
Apparently the initial state wasn't zero temperature. So need to manufacture a zero temperature state.